app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "mSupplyMobile",
+  "displayName": "mSupplyMobile"
```

Comment:
Should this have a space in it? I presume this is what shows under the icon on a device.

Reply:
Yes displayname should have space. But this json file is not being used right now. We should add variable from this json file in places where names are hard coded right now but for now I did not touch the code unless it was absolutely necessary.